server/config/config.js

```diff
@@ -0,0 +1,27 @@
+import dotenv from 'dotenv';
+
+dotenv.config();
+
+module.exports = {
```

Comment:
Please, I need clarification on why do we need to set database variables like username and password when we can just use a database URL for PostgreSQL.